template.jinja2

```diff
@@ -12,6 +12,18 @@
 					({{ frame.filepath }}:{{ frame.line_no}})
 				</p>
 				<code>{{ frame.rendered or '' }}</code>
+				<table class="local-vars">
+					<tr>
+						<th>Local variable</th>
+						<th>Value</th>
+					</tr>
+					{% for key, value in frame.local_vars.items() %}
+						<tr>
+							<td>{{ key }}</td>
+							<td>{{ value }}</td>
```

Comment:
so this is like the actual data in the variable, right? ints are ints, lists are lists, strs are strs? and jinja will... `str()` them? maybe? I think we want to `repr` here if possible. I'm curious what happens if a value is `'\x00'` or something

Reply:
![screenshot 6](https://user-images.githubusercontent.com/1896456/31799624-ce125a1c-b500-11e7-93bd-6e0c36cb11f1.png)